shortcut_test.go

```diff
@@ -15,6 +15,22 @@ func TestShortcutHandler_AddShortcut(t *testing.T) {
 	assert.Equal(t, 2, len(handle.entry))
 }
 
+func TestShortcutHandler_RemoveShortut(t *testing.T) {
+	handle := &ShortcutHandler{}
+	handle.AddShortcut(&ShortcutCopy{}, func(shortcut Shortcut) {})
+	handle.AddShortcut(&ShortcutPaste{}, func(shortcut Shortcut) {})
+
+	assert.Equal(t, 2, len(handle.entry))
+
+	handle.RemoveShortcut(&ShortcutCopy{})
+
+	assert.Equal(t, 1, len(handle.entry))
+
+	handle.RemoveShortcut(&ShortcutPaste{})
+
+	assert.Equal(t, 0, len(handle.entry))
```

Comment:
```suggestion
	handler := &ShortcutHandler{}
	handler.AddShortcut(&ShortcutCopy{}, func(shortcut Shortcut) {})
	handler.AddShortcut(&ShortcutPaste{}, func(shortcut Shortcut) {})

	assert.Equal(t, 2, len(handler.entry))

	handler.RemoveShortcut(&ShortcutCopy{})

	assert.Equal(t, 1, len(handler.entry))

	handler.RemoveShortcut(&ShortcutPaste{})

	assert.Equal(t, 0, len(handler.entry))
```

Reply:
Note, this change is inconsistent with the existing tests in this file